crates/rolldown/src/ecmascript/format/cjs.rs

```diff
@@ -22,17 +22,7 @@ pub fn render_cjs(
   let mut concat_source = ConcatSource::default();
 
   // Add `use strict` directive if needed. This must come before the banner, because users might use banner to add hashbang.
-  let are_modules_all_strict = ctx
```

Comment:
cc @hyf0 The rollup `banner` should be prepend at `use strict` for cjs, you could see [amd example]( https://rollupjs.org/repl/?version=4.19.0&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQSUyMjAxJTIyJTJDJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMmNvZGUlMjIlM0ElMjIlMkYlMkYlMjBOQU1FRCUyMEVYUE9SVFMlNUNuJTJGJTJGJTIwVGhlcmUlMjBhcmUlMjBtYW55JTIwd2F5cyUyMHRvJTIwZXhwb3J0JTIwYmluZGluZ3MlNUNuJTJGJTJGJTIwZnJvbSUyMGFuJTIwRVMyMDE1JTIwbW9kdWxlJTVDbmV4cG9ydCUyMHZhciUyMGZvbyUyMCUzRCUyMDElM0IlNUNuJTVDbmV4cG9ydCUyMGZ1bmN0aW9uJTIwYmFyKCklMjAlN0IlNUNuJTVDdCUyRiUyRiUyMHRyeSUyMGNoYW5naW5nJTIwdGhpcyUyMHRvJTIwJTYwZm9vJTJCJTJCJTYwJTVDbiU1Q3QlMkYlMkYlMjB3aGVuJTIwZ2VuZXJhdGluZyUyMENvbW1vbkpTJTVDbiU1Q3RyZXR1cm4lMjBmb28lM0IlNUNuJTdEJTVDbiU1Q25mdW5jdGlvbiUyMGJheigpJTIwJTdCJTVDbiU1Q3RyZXR1cm4lMjBiYXIoKSUzQiU1Q24lN0QlNUNuJTVDbmV4cG9ydCUyMCU3QiUyMGJheiUyMCU3RCUzQiU1Q25leHBvcnQlMjAqJTIwZnJvbSUyMCcuJTJGcXV4JyUzQiUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTJDJTIybmFtZSUyMiUzQSUyMm1haW4uanMlMjIlN0QlMkMlN0IlMjJjb2RlJTIyJTNBJTIyZXhwb3J0JTIwY29uc3QlMjBxdXglMjAlM0QlMjAnUVVYJyUzQiUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSUyQyUyMm5hbWUlMjIlM0ElMjJxdXguanMlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyb3V0cHV0JTIyJTNBJTdCJTIyZXNNb2R1bGUlMjIlM0ElMjJpZi1kZWZhdWx0LXByb3AlMjIlMkMlMjJleHBvcnRzJTIyJTNBJTIyYXV0byUyMiUyQyUyMmZvcm1hdCUyMiUzQSUyMmFtZCUyMiU3RCU3RCU3RA==)

Reply:
I will modify #1704 when the PR is merged, and fix the banner in it.

Reply:
@7086cmd Could you open another PR for this problem?

Reply:
https://github.com/rolldown/rolldown/issues/1706

Reply:
See #1707.